packages/expo-build-properties/src/ios.ts

```diff
@@ -14,6 +14,11 @@ export const withIosBuildProperties = createBuildPodfilePropsConfigPlugin<Plugin
       propName: 'ios.useFrameworks',
       propValueGetter: (config) => config.ios?.useFrameworks,
     },
+    {
+      propName: 'ios.flipper',
+      propValueGetter: (config) =>
+        typeof config.ios?.flipper === 'string' ? config.ios.flipper : undefined,
+    },
```

Comment:
How does this work when a user sets `flipper` to `true`? Wouldn't it set `'ios.flipper'` to undefined?

Reply:
🐛 Remanent of changing from "enabled" / "disabled" to a boolean. It should follow the design used for `newArchEnabled`. Fixup will include this change, plus altering the Podfile's checks to look at `'true'` and `'false'`.

May require an additional test for string vs boolean config settings; right now the code uses the same snapshot design as the other build properties.